docs/nut-command/setup/setup-deploy.md

```diff
@@ -5,20 +5,30 @@ level: intermediate
 setup:deploy
 ============
 
-Nut's `setup:deploy` is a simple tool to deploy a build of the current site 
+<p class="note"><strong>Note:</strong> The command is provided by a separate
+package called <code>bolt/simple-deploy</code> that is installed by default
+with archive distributions, but needs to manually be included with Composer
+project installs.</p>
+
+Nut's `setup:deploy` is a simple tool to deploy a build of the current site
 from a local workstation to a (S)FTP enabled destination host.
 
 To use this tool, on your development machine **only** you can create a file
 named `.deploy.yml`.
 
+<p class="note"><strong>Note:</strong> This command will not appear in the list
+of available commands if the <code>.deploy.yml</code> file is not present.</p>
```

Comment:
That's what I was missing... Thanks for clarifying!

Reply:
Yeah, massive facepalm on my part